### DISCUSSION TITLE
Radius of nuclear bombs, relative to the crater

### DISCUSSION THREAD

**Discussion by HollyEreTic** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1772] [General]
Am I the only one who thinks that the depth of the nuclear radius of all cores is understated and the crater is slightly exaggerated?, I suggest reducing the crater radius and depth a bit, and increasing the radiation contamination radius by one and a half, 2 times, for better immersion in the atmosphere 
 wasteland scorched by radiation
What do you think about this?
\
\
P.S.I wrote through a translator, so there may be mistakes

**Comment by HjonkBonk** [comment]
Yeah, the hole that the nuclear bomb makes is a bit, well, not really deep enough. However, I think that the radius is perfectly fine. In fact, it is small compared to an actual nuclear bomb.
There is a good reason for the radioactivity levels though, although I would have to go far back into the issues in this repo to find it.
Im sure one of the devs can explain.

**Reply by HollyEreTic** [reply]
Thank you for answering)
CHANGED
although yep,the craters are quite normal, in such a case one would simply like a larger radius of radiation contamination
